release.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-docker build -t indeedoss/rabbitmq-operator:latest .
+docker build -t indeedoss/rabbitmq-operator:latest rabbitmq-operator
```

Comment:
the Dockerfile should probably move into the rabbitmq-operator directory since that's all it builds against

Reply:
I did move the Dockerfile, back in 3e33903.  I just missed updating this until I saw the Travis test build fail.

Reply:
Or did you mean that this file should be moved into the rabbitmq-operator directory as well?  That would require changes to the Travis configuration (I think).